appveyor.yml

```diff
@@ -24,6 +24,8 @@ before_build:
         $all_packages += $packages[$i] + ":" + $env:PLATFORM + "-windows-static "
     }
 
+    git -C C:\Tools\vcpkg pull
```

Comment:
Is it possible to add a trailing comment to this line, explaining that this is only temporary?